src/components/Todo.js

```diff
@@ -0,0 +1,30 @@
+import { StyledTodo } from "./styles/Todo.styled";
+import {Edit2Outline} from '@styled-icons/evaicons-outline/Edit2Outline';
+import {DeleteBin6} from '@styled-icons/remix-line/DeleteBin6';
+
+export default function Todo({text, todo, todos, setTodos}) {
+    const deleteHandler = () => {
+        setTodos(todos.filter(item => item.id !== todo.id));
+    }    
+    const completeHandler = () => {
+        setTodos(todos.map((item) => {
+            if(item.id === todo.id) {
+                return {
+                    ...item, completed: !item.completed
+                }
+            }
+            return item;
+        }))
+    }
+    return (
+        <StyledTodo>
+            <td className={`todo-item ${todo.completed ? "completed" : ""}`}>{text}</td>
```

Comment:
Another way to do conditionals is to create  a styled component and parse todo.completed as a prop. Then you can do all the logic inside the styled component which is quite nice :)